bf_switchd/switch_config.c

```diff
@@ -407,7 +407,13 @@ static void switch_p4_pipeline_config_init(const char *install_dir,
             cJSON *pipe_scope_obj = cJSON_GetArrayItem(pipe_scope_arr, s);
             p4_pipeline->pipe_scope[s] = pipe_scope_obj->valueint;
           }
-        }
+	  cJSON *ct_timeout_arr = cJSON_GetObjectItem(p4_pipeline_obj, "ct_timeout");
```

Comment:
Please Add NULL check, before de-referring, I hope ct_timeout is not mandatory parameter

Reply:
Please verify scenario where .conf file not having ct_timeout  parametere